pkg/kubelet/cm/cpumanager/topology_hints_test.go

```diff
@@ -477,7 +480,8 @@ func TestGetPodTopologyHintsWithPolicyOptions(t *testing.T) {
 				sourcesReady:      &sourcesReadyStub{},
 			}
 
-			podHints := m.GetPodTopologyHints(&testCase.pod)[string(v1.ResourceCPU)]
+			_, ctx := ktesting.NewTestContext(t)
+			podHints := m.GetPodTopologyHints(ctx, &testCase.pod)[string(v1.ResourceCPU)]
```

Comment:
When we start passing the context into these various functions, there can be side-effects (context cancelled, timed out, etc), thus necessitating an err being returned from the functions and handled.

Reply:
even before there were side effects like the call to Device Plugin failed. But you are correct, this needs to be handled.

cc: @ffromani

Reply:
Agreed. Can we start passing context around with a `context.TODO()` at the very top to remind that the wiring is properly done? chasing up all the error paths is the correct thing to do, but won't be trivial.